share/spice/deb_version/deb_version.js

```diff
@@ -0,0 +1,47 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_deb_version = function(api_result_str){
+        var api_result = JSON.parse(api_result_str);
+
+        if (api_result.error) {
+            return Spice.failed('deb_version');
+        }
+
+        Spice.add({
+            id: "deb_version",
+            name: "Debian Package Versions (" + api_result.package + ")",
+            data: api_result,
+            meta: {
+                sourceName: "packages.debian.org",
+                sourceUrl: 'https://packages.debian.org/search?searchon=names&keywords=' + api_result.package
+            },
+            normalize: function(result) {
+                var data = {
+                    record_data: {},
+                };
+
+                for (var i = result.versions.length - 1; i >= 0; i--) {
+                    var version = result.versions[i];
+
+                    for (var j = version.suites.length - 1; j >= 0; j--) {
+                        data.record_data[version.suites[j]] = version.version;
+                    };
+                };
+
+                if (Object.keys(data.record_data).length === 0) {
+                    return Spice.failed('deb_version');
+                }
+
+                return data;
+            },
+            templates: {
+                group: 'base',
```

Comment:
Let's use the `text` template -- base is to be avoided unless absolutely necessary for a custom tile.